fe/fe-core/src/main/java/com/starrocks/catalog/RangePartitionInfo.java

```diff
@@ -48,12 +54,25 @@
 public class RangePartitionInfo extends PartitionInfo {
     private static final Logger LOG = LogManager.getLogger(RangePartitionInfo.class);
 
+    @SerializedName(value = "partitionColumns")
     private List<Column> partitionColumns = Lists.newArrayList();
     // formal partition id -> partition range
     private Map<Long, Range<PartitionKey>> idToRange = Maps.newHashMap();
     // temp partition id -> partition range
     private Map<Long, Range<PartitionKey>> idToTempRange = Maps.newHashMap();
 
+    // partitionId -> serialized Range<PartitionKey>
+    // because Range<PartitionKey> and PartitionKey can not be serialized by gson
```

Comment:
```
because Range<PartitionKey> and PartitionKey can not be serialized by gson
```
why? I think you could change it.

Reply:
Range is defined in guava. and I can not modify it. And more, If I add another data structure to replace Range, all partition related logic should be modified. So I use the method in this pr, solve this problem when serialization and deserialization.